2024-11-11 22:30:32.914 [info] Experiment 'pythonRecommendTensorboardExt' is active
2024-11-11 22:30:32.914 [info] Experiment 'pythonTerminalEnvVarActivation' is active
2024-11-11 22:30:32.914 [info] Experiment 'pythonTestAdapter' is active
2024-11-11 22:30:32.914 [warning] Dir "/home/yoshee/hse/machine_learning/.pixi/envs" is not watchable (directory does not exist)
2024-11-11 22:30:32.914 [info] Default formatter is set to esbenp.prettier-vscode for workspace /home/yoshee/hse/machine_learning
2024-11-11 22:30:32.914 [info] Starting Environment refresh
2024-11-11 22:30:32.914 [info] Searching for interpreters in posix paths locator
2024-11-11 22:30:32.914 [info] Searching for pyenv environments
2024-11-11 22:30:32.914 [info] Searching for conda environments
2024-11-11 22:30:32.914 [info] Searching for global virtual environments
2024-11-11 22:30:32.914 [info] Searching for custom virtual environments
2024-11-11 22:30:32.914 [info] pyenv is not installed
2024-11-11 22:30:32.914 [info] Finished searching for pyenv environments: 21 milliseconds
2024-11-11 22:30:32.914 [info] Finished searching for custom virtual envs: 21 milliseconds
2024-11-11 22:30:32.914 [info] > conda info --json
2024-11-11 22:30:32.914 [info] > pixi --version
2024-11-11 22:30:32.914 [info] > hatch env show --json
2024-11-11 22:30:32.914 [info] cwd: .
2024-11-11 22:30:32.914 [info] Python interpreter path: /bin/python
2024-11-11 22:30:32.914 [info] Finished searching for global virtual envs: 226 milliseconds
2024-11-11 22:30:32.921 [info] > pyenv which python
2024-11-11 22:30:32.921 [info] cwd: .
2024-11-11 22:30:32.948 [info] Found: /bin/python --> /bin/python3.12
2024-11-11 22:30:32.954 [warning] could not find a pixi interpreter for the interpreter at /bin/python
2024-11-11 22:30:32.961 [info] Found: /bin/python3 --> /bin/python3.12
2024-11-11 22:30:32.973 [info] Found: /bin/python3.12 --> /bin/python3.12
2024-11-11 22:30:32.975 [warning] could not find a pixi interpreter for the interpreter at /bin/python
2024-11-11 22:30:32.986 [info] > /bin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:32.993 [info] Found: /sbin/python --> /sbin/python3.12
2024-11-11 22:30:32.993 [info] Found: /sbin/python3 --> /sbin/python3.12
2024-11-11 22:30:32.993 [info] Found: /sbin/python3.12 --> /sbin/python3.12
2024-11-11 22:30:32.998 [info] Found: /usr/bin/python --> /usr/bin/python3.12
2024-11-11 22:30:32.998 [info] Found: /usr/bin/python3 --> /usr/bin/python3.12
2024-11-11 22:30:32.998 [info] Found: /usr/bin/python3.12 --> /usr/bin/python3.12
2024-11-11 22:30:33.030 [info] Found: /usr/sbin/python --> /usr/sbin/python3.12
2024-11-11 22:30:33.031 [info] Found: /usr/sbin/python3 --> /usr/sbin/python3.12
2024-11-11 22:30:33.031 [info] Found: /usr/sbin/python3.12 --> /usr/sbin/python3.12
2024-11-11 22:30:33.033 [info] Found: /home/linuxbrew/.linuxbrew/bin/python3 --> /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.6/bin/python3.12
2024-11-11 22:30:33.033 [info] Found: /home/linuxbrew/.linuxbrew/bin/python3.12 --> /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.6/bin/python3.12
2024-11-11 22:30:33.045 [info] > /sbin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.055 [info] > /usr/bin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.061 [info] Finished searching for interpreters in posix paths locator: 450 milliseconds
2024-11-11 22:30:33.065 [info] Environments refresh paths discovered (event): 454 milliseconds
2024-11-11 22:30:33.065 [info] Environments refresh paths discovered: 454 milliseconds
2024-11-11 22:30:33.083 [info] > /usr/sbin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.095 [info] > /home/linuxbrew/.linuxbrew/bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.154 [info] Environments refresh finished (event): 543 milliseconds
2024-11-11 22:30:33.158 [info] Environment refresh took 547 milliseconds
2024-11-11 22:30:33.160 [error] Environment /bin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.160 [error] Environment /usr/sbin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.160 [error] Environment /sbin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.161 [error] Environment /usr/bin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.186 [info] > /usr/sbin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.191 [info] > /usr/bin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.227 [info] > /sbin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.228 [error] Environment /usr/sbin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.236 [info] > /bin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.238 [error] Environment /usr/bin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.260 [info] > /sbin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.261 [error] Environment /sbin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.270 [info] > /usr/sbin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.271 [error] Environment /bin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.296 [info] > /usr/bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.297 [error] Environment /sbin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.312 [info] > /bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-11 22:30:33.313 [error] Environment /usr/sbin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.330 [error] Environment /usr/bin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.343 [error] Environment /bin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-11 22:30:33.363 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-11 22:30:33.371 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-11 22:30:33.955 [info] Starting Pylance language server.
2024-11-11 22:30:35.258 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-11 22:44:09.429 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-11 22:44:09.434 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-11 22:49:05.711 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
